src/components/propositionalLogic/examplePage.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import styles from "./propositionalLogic.module.scss";
+
+class ExamplesPropositionalLogic extends React.Component {
+	public render() {
+		return <div className={styles.site}>
+			<div className={styles.proofEditor}>
+				<p><button className={styles.buttons}
+					onClick={this.showElement}>
+					Beispiel 1
```

Comment:
This should be a more descriptive name. Possible what statement is being proven in this example.